aten/src/ATen/native/hammerblade/BinaryOpsKernel.cpp

```diff
@@ -25,7 +25,7 @@ void mul_kernel_hb(TensorIterator& iter) {
   if (iter.dtype() == ScalarType::Float) {
     offload_op_binary(iter, 1.0, "tensorlib_mul");
   } else {
-    AT_ERROR("HammerBlade only supports adding two floats");
+    AT_ERROR("HammerBlade only supports multiplying two floats");
```

Comment:
I was wondering if we can use a function definition macro to factor repetitive code...

Reply:
I would like to avoid over using macros. At least until we know more about what we want to do. We can potentially use real C++ metaprogramming to help avoid boiler plate code.

Reply:
I agree, macro are be used sparingly. In this particular scenario though, we need functions with very similar definitions and different _names_. No sure if there's a C++ construct to parameterize function names... this seems to be fundamentally a preprocessing problem.